ee/docker-ee/centos.md

```diff
@@ -100,54 +94,55 @@ $ sudo yum remove docker \
                   docker-engine
 ```
 
-## Repo install and upgrade
+## Install Docker Engine - Enterprise
+
+Docker Engine - Enterprise can be installed either via YUM repositories, or by downloading and installing the RPM package and thereafter manually managing all upgrades. The Docker repository method is recommended, for the ease it lends in terms of both installation and upgrade tasks. The more manual RPM package approach, however, is useful in certain situations, such as installing Docker on air-gapped system that have no access to the Internet.
```

Comment:
don't think you need a comma after "recommended". Also wonder if you need "thereafter".